src/engine/input/key_mapping/InputActionItem.cs

```diff
@@ -0,0 +1,139 @@
+using System;
+using System.Collections.ObjectModel;
+using System.Collections.Specialized;
+using Godot;
+
+/// <summary>
+///   Defines one specific input.
```

Comment:
Isn't this for showing or editing?

Reply:
what do you mean?

Reply:
I mean does this *literally* define one specific input. Or is this a Control for showing / editing a key input?

Reply:
InputActionItem itself does not handle input key editing, but it has InputEventItems as children, which handle input key editing

Reply:
Could you clarify that in the doc. My goal is that if you just read the summaries of all the new classes, you get some kind of picture where abouts different input changing stuff happens.